syntax = "proto3";

// option java_multiple_files = true;
// option java_package = "com.superset.grpc";
// option java_outer_classname = "WesternMovies";

package westernmovies;

// The western movie service definition.
service WesternMovies {
  // Sends a greeting
  rpc GetByName (NameRequest) returns (MovieResponse) {}
}

// The request message asking for details of a movie
message NameRequest {
  string name = 1;
}

// The response message containing the details of the requested Movie
message MovieResponse {
  string year = 1;
  string topCast = 2;
  string overview = 3;
}
